tools/downloader/pytorch_to_onnx.py

```diff
@@ -11,6 +11,18 @@
 import torch.onnx
 
 
+INPUT_DTYPE_TO_TORCH = {
+        'float': torch.float,
+        'half': torch.half,
+        'double': torch.double,
+        'uint8': torch.uint8,
+        'int8': torch.int8,
+        'int': torch.int32,
+        'short': torch.short,
+        'long': torch.long,
+        'bool': torch.bool
```

Comment:
Please reindent with 4 spaces, sort and add a trailing comma.

Reply:
updated